### PR TITLE
[VL] minor fix for velox cache

### PR DESCRIPTION
**Pull request by zhli1142015**
## What changes were proposed in this pull request?
We observed performance gain for velox cache (`CacheBufferInput`  class) with config values specified in https://github.com/oap-project/gluten/commit/b57f56030eef2496e8d871c8f0e766b96661da25.
Here are some test results for 1TB TPCDS (3 runs eache query get fareast one as result), 8 executors(8 cores 56Gb memory each):
Velox cache enabled (mem only 8GB): 4.9%.
Velox cache enabled (mem only 8GB) + soft affinity: 9.7%.
```
"spark.gluten.sql.columnar.backend.velox.cacheEnabled": "true",
"spark.gluten.sql.columnar.backend.velox.memCacheSize": "8589934592",
"spark.gluten.sql.columnar.backend.velox.ssdCacheSize": "0",
"spark.gluten.sql.columnar.backend.velox.fileHandleCacheEnabled": "true",
"spark.gluten.soft-affinity.enabled": "true",
```

The limit of this approach is now the memory used by velox cache is not managed by spark.

### CODE REVIEW

**Comment by zhli1142015** [comment]
cc @zhouyuan

File: gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala [hunk 1]
```diff
@@ -180,6 +180,15 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
       conf.set("spark.sql.orc.enableVectorizedReader", "false")
       conf.set("spark.sql.inMemoryColumnarStorage.enableVectorizedReader", "false")
     }
+
+    if (
```
Comment (zhouyuan) [inline gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala:184]: would be better if you could add a note here

**Review by zhouyuan — APPROVED** [review]
@zhli1142015 
thanks for the fix! The local cache has been broken for a while as i know.

File: gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala (hunk 1 above)
Comment (zhli1142015) [reply gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala:185]: Added, thanks

**Review by zhouyuan — APPROVED** [review]
👍 
works for me

**Pull request merged**